runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/Bindings.java

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Please rename this to `BindingsLayout` and move to `layouts` subpackage.

Reply:
OK